.travis.yml

```diff
@@ -1,14 +1,26 @@
+# sudo required for installing dependencies of Scala Native
+sudo: required
 language: scala
-scala:
-  - 2.10.7
-  - 2.11.12
-  - 2.12.4
-  - 2.13.0-M3
```

Comment:
Unfortunately the build matrix is no longer defined for every possible scala-platform combination (native only supports 2.11, whereas we build spray on the JVM for 2.10, 2.11, 2.12 and 2.13).
The adaptations I made to the build script will still build and test spray for every possible version combination, however it will no longer happen with travis sub-tasks. Instead, there is one task that instructs sbt to build and test against any versions defined in its build configuration.